scripts/normalize_user_tags.py

```diff
@@ -90,20 +90,24 @@
 def migrate_source_tags(tags):
     Tag = apps.get_model('osf', 'Tag')
     for tag_name in tags:
-        try:
-            tag = Tag.all_tags.get(name=tag_name[0], system=True)
+        tag, created = Tag.all_tags.get_or_create(name=tag_name[0], system=True)
```

Comment:
Well if you do it this way, all the old source tags will exist! you either fetched the existing one, or created one. Then you can just rename them all!

This solution could leave old source tags lying around, because the `created` ones won't be renamed.